### ISSUE TITLE
Skin editor not opening

### ISSUE THREAD

**Issue by winkelnp** [flathub/com.mojang.Minecraft#110]
The Flatpak version does not allow the skin editor to open, while the official AUR one does. Would it be possible to get this fixed? I feel like especially a microsoft product needs the extra sandboxing.

**Comment by AsciiWolf** [comment]
You mean the "Skins" tab in Launcher? It [did not work](https://github.com/flathub/com.mojang.Minecraft/issues/45) for some time, but should work fine now.

**Comment by winkelnp** [comment]
No, not the "skins" tab, when you click the "edit" button on the individual skins

**Comment by AsciiWolf** [comment]
Even this seems to work fine on my system (with Intel GPU). I will try it on my another system (with AMD GPU) tomorrow.

**Comment by winkelnp** [comment]
It might be an AMD thing, I'm running one

**Comment by AsciiWolf** [comment]
I can reproduce it on my other machine with AMD GPU. When I click "Edit", nothing happens. There is nothing in log and it works fine on my Intel GPU laptop. Anyway, most likely is not a Flatpak-related issue and you will have the same problem with non-Flatpak'ed version of Minecraft Launcher.

**Comment by winkelnp** [comment]
As I mentioned earlier, the AUR version of the launcher does not have this problem for me.

**Comment by AsciiWolf** [comment]
Maybe it is related to the Mesa version included in the Freedesktop runtime used by Minecraft Flatpak.

**Comment by AsciiWolf** [comment]
Can you please retest it with the latest freedesktop runtime that got released recently? Thanks.

**Comment by winkelnp** [comment]
I have everything updated to the newest version and the issue persists

**Comment by vchernin** [comment]
I've had trouble with the edit button within the installations panel. To workaround just clicking the name of the installation seems to somehow work.

Perhaps that trick is also applicable to skins?

**Comment by winkelnp** [comment]
that is:
org.freedesktop.Platform 21.08.7
org.freedesktop.Platform.GL.default 21.3.1
